cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -168,6 +168,18 @@
         overflow: visible !important;
     }
 
+    .highcharts-tooltip > span {
+        z-index: 9999;
+        padding: unit( 4px / @base-font-size-px, em )
+                 unit( 10px / @base-font-size-px, em )
+                 unit( 4px / @base-font-size-px, em )
+                 unit( 8px / @base-font-size-px, em );
+        border: 2px solid @green-60;
+        margin: 0 2px;
+        background-color: #fff;
```

Comment:
Could use `@white` so colors are always variables.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4047/commits/805de5edd2f3f85ba299ecef445c033fc1e117a9.